src/Field/AbstractType.php

```diff
@@ -47,9 +47,13 @@ public function render(array $attributes = [], array $viewData = [])
     {
         $this->options = array_merge($this->options, $attributes);
 
-        $value = request($this->name, $this->getOption('value'));
-
-        $this->setValue( old($this->name, $value) );
+        if (func_num_args() == 3) {
```

Comment:
This if statement checks for an optional third parameter. If it exists, it assumes that's the value and sets it manually.

func_get_arg(2) actually gets the third parameter because it works with indexes starting from 0.

The issue itself is described in the PR. I've tested this code and it works